### PR TITLE
amazon_reviews_multi:clean and add new prompt

### PR DESCRIPTION
**Pull request by sbmaruf**
Added two new prompt since the [original paper](https://arxiv.org/pdf/2010.02573.pdf) categorize the task based on review `Body, title & category` 

Other Metric refers to MAE mentioned in the paper.

Please review the `answer_choices` in the template.

### CODE REVIEW

File: promptsource/templates/amazon_reviews_multi/en/templates.yaml [hunk 1]
```diff
@@ -4,21 +4,22 @@ templates:
...
 
       {{review_title}}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
-      original_task: null
-    name: Template_2
+      choices_in_prompt: false
+      metrics:
+      - BLEU
```
Comment (VictorSanh) [inline promptsource/templates/amazon_reviews_multi/en/templates.yaml:17]:
```suggestion
      - BLEU
      - ROUGE
```

File: promptsource/templates/amazon_reviews_multi/en/templates.yaml [hunk 2]
```diff
@@ -44,42 +47,86 @@ templates:
...
       {{product_category}}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
-      original_task: null
-    name: Template_1
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      - BLEU
```
Comment (VictorSanh) [inline promptsource/templates/amazon_reviews_multi/en/templates.yaml:92]:
```suggestion
      - Other
```
BLEU is so weird for unigrams... removing that
also for the record, accuracy might be non optimal either (there are 5'000 categories in the test set alone)

File: promptsource/templates/amazon_reviews_multi/en/templates.yaml [hunk 3]
```diff
@@ -44,42 +47,86 @@ templates:
...
     reference: Product category based on review body
+  8e8973f6-431f-4e78-b83a-a86c04655882: !Template
+    answer_choices: 1|||2|||3|||4|||5
+    id: 8e8973f6-431f-4e78-b83a-a86c04655882
+    jinja: 'Rate the product by the number of stars based on the review below: (1
+      being the lowest and 5 the highest)
+
+      ===
+
+      {{review_title}}. {{review_body}}, Product category: {{product_category}}|||
```
Comment (VictorSanh) [inline promptsource/templates/amazon_reviews_multi/en/templates.yaml:104]:
```suggestion
      {{review_title}}. {{review_body}}
      Product category: {{product_category}}|||
```

File: promptsource/templates/amazon_reviews_multi/en/templates.yaml [hunk 4]
```diff
@@ -44,42 +47,86 @@ templates:
...
+      - Accuracy
+      - Other
+      original_task: true
+    name: prompt_body_title_category_to_star
+    reference: Rating based on review body, title, category
   c4717e75-4d3e-4b79-9737-167155f51513: !Template
     answer_choices: null
     id: c4717e75-4d3e-4b79-9737-167155f51513
-    jinja: 'Guess the product category from the below review title:
+    jinja: 'Guess the product category from the review title:
```
Comment (VictorSanh) [inline promptsource/templates/amazon_reviews_multi/en/templates.yaml:118]:
```suggestion
    jinja: 'Guess the product category from the review title below:
```

File: promptsource/templates/amazon_reviews_multi/en/templates.yaml [hunk 5]
```diff
@@ -44,42 +47,86 @@ templates:
...
       {{product_category}}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
-      original_task: null
-    name: Template_3
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      - BLEU
```
Comment (VictorSanh) [inline promptsource/templates/amazon_reviews_multi/en/templates.yaml:129]:
```suggestion
      - Other
```
same as above

**Review by VictorSanh — APPROVED** [review]
a few changes
BLEU is extremely weird for unigrams so I removed it

**Comment by sbmaruf** [comment]
Thanks a lot @VictorSanh for making the necessary changes.

**Pull request merged**